portfolio/src/main/webapp/comments/CommentFormRenderer.js

```diff
@@ -0,0 +1,33 @@
+class CommentFormRenderer extends RendererInterface {
+    constructor(targetElement) {
+        super();
+
+        this.targetElement = targetElement;
+    }
+
+    render(user) {
+        if(user) {
```

Comment:
It is user friendly to hide/show elements based on users status, e.g. to not show new comment form to user who hasn't logged in. But in general must be also handled on the server side.

Reply:
Well, the user parameter (now changed to userEmail for clarity) is the user's email. Added clarification JS Doc to avoid confusion on the future!

Reply:
Please ignore the above comment! I changed the parameter to "hasUser" on the render. The checking happens on the CommentFacade

Reply:
And I will add permission checking on the backend on the next PR!